src/ORM/Connect/DBQueryBuilder.php

```diff
@@ -57,9 +61,73 @@ public function buildSQL(SQLExpression $query, &$parameters)
                 "Not implemented: query generation for type " . get_class($query)
             );
         }
+
+        if (Director::isDev()) {
+            $sql = $this->buildTraceComment() . $sql;
+        }
+
         return $sql;
     }
 
+    /**
+     * Builds an SQL comment indicating where the query was executed from.
+     */
+    protected function buildTraceComment(): string
+    {
+        $comment = '/* ';
+
+        $ignoreBaseClasses = [
```

Comment:
Just put a comment in and/or rename the var to make it clear that these aren't excluded, it's just we go 'further up' in order to work out what called it

Reply:
Done